app/src/main/res/layout/view_image_title_description.xml

```diff
@@ -1,43 +1,43 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/imageTitleDescriptionContainer"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
`xmlns:app="http://schemas.android.com/apk/res-auto"` can be removed.

Reply:
OK, but it's very annoying the next time I want to add an `app:` attribute, and I have to hunt down the correct `xmlns` statement to put at the top of the file. Can we think about having these declarations in each XML file by default?

Reply:
Make sense to me to have the statement by default.